### PR TITLE
fix(projects list): deactivating-a-project-not-possible (DEV-3206)

### PR DESCRIPTION
**Pull request by domsteinbach**
Fix deactivating as well as reactivating

resolves DEV-3206

### CODE REVIEW

File: apps/dsp-app/src/app/system/projects/projects-list/projects-list.component.ts [hunk 1]
```diff
@@ -161,22 +161,30 @@ export class ProjectsListComponent implements OnInit, OnDestroy {
...
 
-    this._projectApiService.update(id, data).pipe(
-      tap(() => {
-        this.refreshParent.emit();
-      })
-    );
+    this._projectApiService
+      .update(id, data)
+      .pipe(
+        take(1),
```
Comment (derschnee68) [inline apps/dsp-app/src/app/system/projects/projects-list/projects-list.component.ts:183]: you can remove take(1) as http observables always complete.

File: apps/dsp-app/src/app/system/projects/projects-list/projects-list.component.ts [hunk 2]
```diff
@@ -161,22 +161,30 @@ export class ProjectsListComponent implements OnInit, OnDestroy {
...
   deactivateProject(id: string) {
-    this._projectApiService.delete(id).pipe(
-      tap(() => {
-        this.refreshParent.emit(); // TODO Soft or Hard refresh ?
-      })
-    );
+    this._projectApiService
+      .delete(id)
+      .pipe(
+        take(1),
```
Comment (derschnee68) [inline apps/dsp-app/src/app/system/projects/projects-list/projects-list.component.ts:167]: you can remove take(1)

Comment (derschnee68) [reply apps/dsp-app/src/app/system/projects/projects-list/projects-list.component.ts:167]: unnamed

**Review by derschnee68 — CHANGES_REQUESTED** [review]

File: apps/dsp-app/src/app/system/projects/projects-list/projects-list.component.ts (hunk 2 above)
Comment (domsteinbach) [reply apps/dsp-app/src/app/system/projects/projects-list/projects-list.component.ts]: Oh, nice. Done.

**Review by derschnee68 — APPROVED** [review]

**Pull request merged**